### PR TITLE
Fixes issue 795 and adds maze's drag and drop images

### PR DESCRIPTION
**Pull request by joeinnes**
Rebased as discussed.

### CODE REVIEW

**Comment by TimohStudios** [comment]
Did you rebase it the changes in your fork? Looks like there's quite a few commits that don't belong in here.

**Comment by TimohStudios** [comment]
Hihi, took me a bit to figure out you meant mazzo's instead of maze's.

**Comment by joeinnes** [comment]
Ah stupid autocorrect! Sorry!

Sent with [inky](http://inky.com?kme=signature)

**Comment by CraigChilds94** [comment]
Please may you tidy this PR up? so that we only have your changes coming through. It will mean that we can easily merge it in.

**Comment by joeinnes** [comment]
Tidied up and submitted new PR.

File: anchor/models/post.php [hunk 1]
```diff
@@ -59,6 +59,15 @@ public static function search($term, $page = 1, $per_page = 10) {
 				Base::table('users.bio as author_bio'),
 				Base::table('users.real_name as author_name')));
 
+		foreach ($posts as $key => $post) {
+			if ($post->data['status'] !== 'published') {
+				unset($posts[$key]);
+			}
+		}
+		if (count($posts) < 1) {
```
Comment (lifehome) [inline anchor/models/post.php]: Why not using $query->count()? And this is repeating the $total, maybe a shorthanded if/else form here to replace the one above?

Comment (joeinnes) [inline anchor/models/post.php]: This isn't my code - I've rebased the PR and resubmitted as #819.

**Pull request closed without merging**